README.markdown

```diff
@@ -44,6 +44,14 @@ Eg: `curl ... | NVM_DIR="path/to/nvm" bash`
 
 Note: On OSX, if you get `nvm: command not found` after running the install script, your system may not have a [.bash_profile file] where the command is set up. Simple create one with `touch ~/.bash_profile` and run the install script again.
 
+### Verify installation
+
+To verify that nvm has been installed, do 
+    
+    nvm --version
+    
+which should output the version number of nvm that you just installed. 
```

Comment:
Maybe add something like "if it does not, try `type nvm` - note that `which nvm` will not work, because `nvm` is a sourced shell function, not an executable binary"?